proxy/InkAPI.cc

```diff
@@ -8256,6 +8265,17 @@ TSHttpTxnConfigStringSet(TSHttpTxn txnp, TSOverridableConfigKey conf, const char
     if (value && length > 0) {
       s->t_state.txn_conf->client_cert_filepath = const_cast<char *>(value);
     }
+  case TS_CONFIG_HTTP_RESPONSE_VIA_TRANSPORT:
+    data._s.setView(value, length);
+    if (ViaTransportVerbosityConversion(s->t_state.txn_conf->response_via_transport, "proxy.config.http.response_via_tranport",
+                                        RECD_STRING, data)) {
+    }
+    break;
+  case TS_CONFIG_HTTP_REQUEST_VIA_TRANSPORT:
+    data._s.setView(value, length);
+    if (ViaTransportVerbosityConversion(s->t_state.txn_conf->request_via_transport, "proxy.config.http.request_via_tranport",
```

Comment:
Yeah, not sure what to do about that. It might well work now to just set it to an int value in a plugin - you can see the parsing code handles direct integers in addition to the strings. If we added this kind of enum to LibRecords that would be easier to do.